src/drawers/drawer.ts

```diff
@@ -168,8 +177,8 @@ export class Drawer {
   /**
    * Returns the D3 selection corresponding to the datum with the specified index.
    */
-  public selectionForIndex(index: number): d3.Selection<any> {
-    return d3.select(this.selection()[0][index]);
+  public selectionForIndex(index: number): SimpleSelection<any> {
```

Comment:
the previous implementation would validate the selection before returning. are we sure we're okay with losing that?